{"name": "Проверка получения списка заказов с невалидным курьером", "status": "passed", "description": "Проверяем получение сообщения об ошибке при использовании случайных значений courierId", "steps": [{"name": "Отправка запроса на получение заказов", "status": "passed", "parameters": [{"name": "courierId", "value": "2852493"}], "start": 1715550828503, "stop": 1715550828647}], "start": 1715550828502, "stop": 1715550828650, "uuid": "4908dbd6-7e4e-4ffb-b666-cc9829a9b9e4", "historyId": "529ddb4e61781ff14c7a8c359eee7c15", "testCaseId": "529ddb4e61781ff14c7a8c359eee7c15", "fullName": "tests.test_get_order.TestGetOrder#test_get_order_with_invalid_courierId", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_order"}, {"name": "subSuite", "value": "TestGetOrder"}, {"name": "host", "value": "MacBook-Pro-Ivan-4.local"}, {"name": "thread", "value": "22077-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_order"}]}